version/distro/distro_test.go

```diff
@@ -13,3 +17,58 @@ func BenchmarkGet(b *testing.B) {
 	}
 	_ = d
 }
+
+func TestParseLinuxOSRelease(t *testing.T) {
+	tests := []struct {
+		desc      string
+		osRelease string
+		want      Distro
+	}{
+		{
+			desc: "regular",
+			osRelease: `
+NAME=Debian
+ID=debian
+`,
+			want: Debian,
+		},
+		{
+			desc: "single quoted",
+			osRelease: `
+NAME=Debian
+ID='debian'
+`,
+			want: Debian,
+		},
+		{
+			desc: "double quoted",
+			osRelease: `
+NAME=Debian
+ID="debian"
```

Comment:
Not sure if it would ever happen, and I think your `Trim` call would handle it, but maybe test unclosed and un-opened quotes?

"Debian
Debian'